modules/statics/src/coins.ts

```diff
@@ -1336,6 +1337,28 @@ export const coins = CoinMap.fromCoins([
     POLYX_FEATURES,
     KeyCurve.Ed25519
   ),
+  account(
+    '98071460-1488-4edd-857f-0899bc5eee4f',
+    'vet',
+    'Mainnet VET',
+    Networks.main.vet,
+    18, // 1 VET = 10^18 wei
+    UnderlyingAsset.VET,
+    BaseUnit.ETH, // The smallest unit of VET, similar to Ethereum, is called 'wei'.
```

Comment:
kind of nitty, but will the enum allow for us to make a different const for VET also under the name `wei`? I just don't want people to think eth and vechain are related when they are each their own L1's

Reply:
No, it won't allow to create another const named as `wei` in the `BaseUnit` enum.
I understand it might be bit confusing hence I put detailed comment.

Also, many other chain configs had `BaseUnit.ETH`.